README.md

```diff
@@ -1,4 +1,4 @@
-![owl logo](https://dvc.org/static/img/logo-github-readme.png)
+![owl logo](https://dvc.org/img/logo-github-readme.png)
```

Comment:
why removing static prefix?

Reply:
Gatsby use /public/static/` folder to store autogenerated images and other assets. I decided that using the same name can cause possible new bugs with name confilcts and can make checking the build results harder, so I removed `/static/` part.

Reply:
Oops, my error. Forgot that `gatsby` actually automatically remove it. We still have folder `/static/img` in repo, but `gatsby` copies it's content to the root of the `/public/` folder. I could add nested folder `/static/static/img`, but I decide that it will be strange.

Reply:
yeah, my concern is to keep as less URLs "locked" as possible. If we can get it into static it would be better (better w/o static/static of course). Also, I was thinking that we apply some image processor like in blog - to unify it, unify plugins, etc?

Reply:
Yep. Perfectly we should move images to corresponding components' folders and import them as other assets, this way will webpack automatically optimize images for us and rename them, move them correctly.

Reply:
to corresponding "md"-files' folders you mean, right?

Reply:
Some to the md files, some to the components (like the icons in `/static/img/community/`).

Reply:
👍

Reply:
@iAdramelk please, create a ticket for this as well

Reply:
Done https://github.com/iterative/dvc.org/issues/1063

Reply:
thanks!